woocommerce/wc-admin
# Add existing TaskList component to new home screen

Progress towards #4096

This PR seeks to render the existing `<TaskList />` component on the new home screen (before modifying it to suit the new designs).

### Accessibility




### Screenshots

<img width="1257" alt="Screen Shot 2020-05-12 at 11 11 12 AM" src="https://user-images.githubusercontent.com/63922/81724266-4fc02900-9441-11ea-9fe3-c6cee74a2b9f.png">

### Detailed test instructions:

- Ensure the task list is enabled
- Open the new home screen
- Verify presence of Task List
- Enter an individual task
- Verify only the task is displayed



### Changelog Note:
